x-pack/plugins/monitoring_collection/server/plugin.ts

```diff
@@ -85,6 +121,55 @@ export class MonitoringCollectionPlugin implements Plugin<MonitoringCollectionSe
     };
   }
 
+  private configureOpentelemetryMetrics(
+    serviceName?: string,
+    serviceInstanceId?: string,
+    serviceVersion?: string
+  ) {
+    const meterProvider = new MeterProvider({
+      resource: new Resource({
+        [SemanticResourceAttributes.SERVICE_NAME]: serviceName,
+        [SemanticResourceAttributes.SERVICE_INSTANCE_ID]: serviceInstanceId,
+        [SemanticResourceAttributes.SERVICE_VERSION]: serviceVersion,
+      }),
+    });
+
+    metrics.setGlobalMeterProvider(meterProvider);
+
+    const otlpConfig = this.config.opentelemetry?.metrics.otlp;
+    const url = otlpConfig?.url ?? process.env.OTEL_EXPORTER_OTLP_METRICS_ENDPOINT;
```

Comment:
`OTEL_EXPORTER_OTLP_ENDPOINT` should be allowable too. It'll be convenient once we align traces and metrics both on the otel spec. Could open that as a follow up issue.

Reply:
I will add the `OTEL_EXPORTER_OTLP_ENDPOINT` - I forgot about it. It's a small change.